spec/models/early_career_payments/eligibility_spec.rb

```diff
@@ -330,23 +330,6 @@
       end
     end
 
-    context "when saving in the 'poor-peformance' context" do
```

Comment:
I think we can remove the validations from the class. They are also run on the submit action but I'm not sure this is necessary.

Reply:
@slorek mmh I removed those [here](https://github.com/DFE-Digital/claim-additional-payments-for-teaching/pull/2666/files#diff-0ae6631c29911caa072ad79b782f98e861fa65cbbdd0cdf8d49ad606c51b1f76)

Ye the submit context is a little bit of a tricky one but I guess we are saying we can get rid of it for now